lib/cli/cliPlugin.js

```diff
@@ -0,0 +1,346 @@
+'use strict';
+const CLI = require('./messages.js');
+const CLICommand = require('./cliCommand.js');
+const { getObjectFrom } = require('./cliTools');
+const tools = require('../tools.js');
+const fs = require('fs');
+const path = require('path');
+
+const controllerIoPackPath = path.join(__dirname, '../../io-package.json');
```

Comment:
nice mixture :-) not as separate params? :)

Reply:
What do you mean? The ../..? I always do it that way. Separate arguments make it less readable IMO.

Reply:
But then you could also just add strings. On windows you end up with different separators mixes up. Path.join will from my knowledge only concats the parts using system path separator. But pats itself stay unchanged.

Reply:
Nope, all path methods normalize the path separators. This is Windows:
```
domin@DESKTOP-OCULUR0 MINGW64 /c/Repositories/node-zwave-js (master)
$ cat src/test.js
const path = require("path");
console.log(__dirname);
console.log(path.join(__dirname, "../package.json"));

domin@DESKTOP-OCULUR0 MINGW64 /c/Repositories/node-zwave-js (master)
$ node src/test.js
C:\Repositories\node-zwave-js\src
C:\Repositories\node-zwave-js\package.json
```

Reply:
See also the [documentation](https://nodejs.org/api/path.html#path_path_join_paths)

> The path.join() method joins all given path segments together using the **platform-specific separator** as a delimiter, then **normalizes** the resulting path.

Reply:
Cool. Again learned something.